src/jace/translator/jaxpr_translator_builder.py

```diff
@@ -721,20 +718,22 @@ class TranslationContext:
     """
 
     sdfg: dace.SDFG
-    inp_names: tuple[str, ...] | None
+    input_names: tuple[str, ...] | None
     out_names: tuple[str, ...] | None
     start_state: dace.SDFGState
     terminal_state: dace.SDFGState
+    jaxpr: jax_core.ClosedJaxpr
```

Comment:
I agree it is a bit useless, but when you debug and would like to check if the translator skewed up it is very helpful if you have it.
I know that because there were some times when I did not had it, or obtaining it was hard.

Reply:
Then please improve the description in the docstring explaining that it's mainly for debugging purposes, because the current description is not very informative.